packages/fxa-content-server/app/scripts/lib/survey-filter.js

```diff
@@ -0,0 +1,240 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UserAgent from './user-agent';
+
+// All but the default export here is to make testing easier.
+
+const falseFn = () => false;
+export const NONE = Symbol();
+
+export const participatedRecently = (
+  previousParticipationTime,
+  doNotBotherSpan
+) => {
+  if (!previousParticipationTime) return true;
```

Comment:
I think that this line should return false

Reply:
I think it needs a stricter check, if the reason for that suggestion is that I passed zeroes in some tests.  That was my bad; I'll fix those tests.  And I'll update this to check for `null` and `undefined`, and to return true on those.

Reply:
Oh actually, those zeroes were for a different param.  Could you explain why you think this should return `false`?  If `surveyTime` didn't exist, it would mean that the user has never taken a survey before.

Reply:
Oh I see the problem now. Cool, thanks for fixing this.  I'll keep the null and undefined check around.